packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -17,7 +17,15 @@ import { captureException } from '@sentry/core';
 import { formGroups } from './formGroups';
 
 const isEAForum = forumTypeSetting.get() === 'EAForum'
-const frontpageDefault = isEAForum ? () => new Date() : undefined
+function forumFrontpageDate (document: Partial<DbPost>) {
```

Comment:
LW is also a forum, so I feel this will accrue ambiguity. Can we always use eaForum?